packages/fscomponents/src/components/AddToCart.tsx

```diff
@@ -31,6 +31,7 @@ export interface AddToCartProps {
   swatchesProps?: any;
   stepperProps?: any;
   buttonProps?: any;
```

Comment:
Did you intend to leave the old any `buttonProps` in here?

Reply:
Definitely didn't. I messed up pulling the changes apart. Fixed.